src/components/Dropdown/Dropdown.scss

```diff
@@ -83,13 +82,13 @@ $border-width: 1px;
 .moonstone-dropdown_outlined {
     color: var(--color-gray_dark);
 
-    box-shadow: inset 0 0 0 $border-width var(--color-gray40);
+    border: var(--selector-border);
 
     &.moonstone-disabled {
         color: var(--color-gray);
 
-        background-color: var(--color-gray_light40);
+        border: var(--selector-border);
```

Comment:
Maybe you don't need to repeat the border here as it is already set with `.moonstone-dropdown_outlined`